### PR TITLE
Initial_update

### PR DESCRIPTION
**Pull request by IamAnkita-create**
little grmmartical mistakes were correted to make look document nice.

### CODE REVIEW

**Comment by IamAnkita-create** [comment]
Please accept as it's the end of hacktoberfest.
Thankyou.

**Review by IamAnkita-create — COMMENTED** [review]
greate work!!

**Comment by koppor** [comment]
Variable names must not take spaces. Thus, this PR should be flagged `invalid` as outlined at https://hacktoberfest.digitalocean.com/faq.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Philately
 
-Philately is a super easy to use rendering engine for HTML5 `<canvas>`. You have a renderer, some layers and that's all. Simple as that. Great for games, interactive media or just drawing some images to the canvas.
```
Comment (hbmueller) [inline README.md:3]: The article here was intended. In the sense of:

> [...] **a** super easy to use **rendering engine** [...]

File: README.md [hunk 2]
```diff
@@ -149,9 +149,9 @@ An object that is passed to the `onStep` function. It contains the following pro
 
 - `addLayer( layer, shouldDraw = true )`: Adds a `Layer` instance the top of the layer stack. By default, draws to the target
   once the layer is ready.
-- `removeLayer( layer, shouldDraw = true )`: Removes a layer from the stack. If the layer was added to the engine multiple
+- `removeLayer( layer, should draw = true )`: Removes a layer from the stack. If the layer was added to the engine multiple
   times, only the first occurrence is removed. By default, updates the target canvas after removing the layer.
-- `draw()`: Draws the layer stack to the target `<canvas>`. Specially useful to update the graphics when the engine `autoStart`
```
Comment (hbmueller) [inline README.md:154]: Good catch.

File: README.md [hunk 3]
```diff
@@ -97,14 +97,14 @@ new Layer({
...
 
-- `sourcePromise`: _[read only]_ Promise that resolves as soon as the source is ready
+- `source promise`: _[read only]_ Promise that resolves as soon as the source is ready
 - `source`: _[read only]_ Reference to the dom element of the source
 - `size`: _[read only]_ An object containing the `width` and `height` of the layer source
 - `posX`: The X position of the layer
 - `posY`: The Y position of the layer
 - `opacity`: A number from 0 to 1 representing the opacity of the layer
 - `isActive`: A boolean for enabling/disabling the layer
-- `onStep`: A function that is called before the each rendering cycle. The parameters passed are [`renderParams`](#renderParams)
```
Comment (hbmueller) [inline README.md:107]: Nice

**Review by hbmueller — CHANGES_REQUESTED** [review]
The names of variables, properties and methods (eg. `myLayer`, `sourcePromise`, `autoResize` and such) can't take spaces or dashes. That's why they are supposed to be in camel case, like they are.

**Comment by hbmueller** [comment]
BTW, @ankitasharma88990 sorry for not reviewing your PR before. For some reason my email notifications were turned off and I only saw this today.

**Comment by hbmueller** [comment]
Closed in favor of #6

**Pull request closed without merging**